build_tools/platforms/BUILD

```diff
@@ -59,3 +77,15 @@ config_setting(
     constraint_values = ["@platforms//os:macos"],
     visibility = ["//visibility:public"],
 )
+
+bool_flag(
```

Comment:
This is left over from when I was trying to do this a nicer way, these do nothing but we may use them in the future